robot_motion_planning_summary.tex

```diff
@@ -183,213 +494,364 @@ \subsubsection{Weighted Randomized Tree Expansion}
+\subsubsection{Kinodynamic RRT}
+For solving problems for systems with differential constraints (like non-holonomic motion constraints), kinodynamic RRT can be used.
+Standard PRM and RRT cannot be applied here.
+Kinodynamic RRT grows the tree respecting the differential constraints.
 
-~\\
-\textbf{Burschka-Style}:
-\begin{enumerate}
-	\item let $F$ be the free space (set of points); $\mu(G)$ the volume/ area of some set G; $reach(x)$ the space ($\subseteq F$) that can be seen from some point $x$; $reach(G)$ the space that can be seen from any point in a set $G$
-	\item find point $x \in F$ with smallest $\mu(reach(x))$
-	\item $S = reach(x)$
-	\item $\epsilon = \frac{\mu(S)}{\mu(F)}$
-	\item choose some set $Z \subseteq S$ that can reach a lot and has an easy to calculate area
-	\item $\alpha = \frac{\mu(Z)}{\mu(S)}$
-	\item $\beta = \frac{\mu(reach(Z) \setminus S)}{\mu(F \setminus S)}$ (not really according to the definition, but best way to do it in the exam)
-	\item the goal is to keep $\alpha$ and $\beta$ balanced
-\end{enumerate}
+\begin{center}
+	\includegraphics[width=0.5\columnwidth]{Kinodynamic_RRT.jpg}
+\end{center}
 
-Using these values, the number n of samples that is needed for a good connectivity can be calculated:
-\[ n= \frac{8 \ln (\frac{8}{\epsilon\alpha\gamma})}{\epsilon\alpha} + \frac{3}{\beta} \,, \] 
-where $1-\gamma$ measures the probability that the uniformly sampled milestones have the correct connectivity ($\gamma \in [0,1]$).
+\subsubsection{RRT*}
+Since RRT (and also PRM) are not optimal, we have to ”re-wire” the graph structure to gain optimality.\\
+Procedure: For each new sample $q_n$, check its $\log n$ neighborhood and if there are better paths from $q_{start}$ to $q_n$, pick that path.\\
+This is an asymptotically optimal sampling based algorithm, because as the number of samples goes to infinity, an optimal path from $q_{start}$ to $q_{goal}$ is obtained.
 
-\subsection{Path and Map Smoothing}
-The paths obtained by random sampling techniques are not necessarily the most efficient paths. Path smoothing techniques are used to optimize the path for length, energy, distance to obstacles etc. 
+\section{Probabilistic Robotics}
+\subsection{Kalman Filter}
+The basic idea of a Kalman filter is that you interpret your robots state as a normal (gaussian) distribution with mean and variance.
+Besides the estimate for the state $x$ you also have some estimate for the uncertainty in your value for $x$ that is represented through a normal distribution and its variance / covariance matrix $P$.
+You also model your sensors with white noise (gaussian, zero-mean).\\
 
-To grow the distance to obstacles, the map can be smoothed by itself before planning. For example, a 2D-convolution with a Gaussian filter kernel leads to a non-zero probability distribution around the edges of the obstacles (edge smoothing).
+\textit{\textbf{Explanantion from the internet:}}
```

Comment:
This heading looks out of place, you might as well remove it. ("from the internet" is not a source)

Reply:
Good point, combined the two paragraphs.